### DISCUSSION TITLE
Fooevent tools

### DISCUSSION THREAD

**Discussion by odaymusic** [inspireui/support#5154] [New Feature/Ideas Requests]
Please add fooevent, as it is compatible with WCFM, and it provides reservation services for events, sports and online courses using zoom, party tickets and seating, and it is a great tool that will provide everything to any user
https://www.fooevents.com

**Comment by gregkeys** [comment]
We would love this as many of our customers want to sell tickets, currently they just add the tickets as a product but if we could have ticket scanning and number generation etc.. things like that would elevate our product to the next level
